rhessys/util/readtag_worldfile.c

```diff
@@ -89,8 +89,9 @@ param *readtag_worldfile(int *paramCnt, FILE *file,char *key){
 	    if(strcmp(strbuf2,"n_basestations")==0){
 	      break;
 	    }
+	    
```

Comment:
Awesome, thanks Brian!

On Mon, Mar 28, 2016 at 7:34 AM, Brian Miles <notifications@github.com>
wrote:

> In rhessys/util/readtag_worldfile.c
> <https://github.com/RHESSys/RHESSys/pull/82#discussion_r57577349>:
>
> > @@ -89,8 +89,9 @@ param *readtag_worldfile(int *paramCnt, FILE *file,char *key){
> >  	    if(strcmp(strbuf2,"n_basestations")==0){
> >  	      break;
> >  	    }
> > +	
>
> Okay, I tested the code on an ~860 MB worldfile and my hypothesis was
> wrong. The new code, as written, is actually quite a bit faster than the
> previous code (~12 second vs. 30 second startup). So don't waste any time
> adding in a more complicated data structure!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/RHESSys/RHESSys/pull/82/files/e0ec292f00e9aaf33834fad68ab9df8d5e9ada61#r57577349>
>



-- 
Xiaoli Chen

Bren School of Environmental Science and Management
University of California, Santa Barbara
xiaoli_chen@umail.ucsb.edu